### PR TITLE
Added device TZE204_ex3rcdha

### PR DESCRIPTION
**Pull request by motezuki**

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -11603,6 +11603,87 @@ const definitions: DefinitionWithExtend[] = [
...
         },
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: '_TZE204_ex3rcdha',
+            },
+        ],
+        model: 'TS0601',
```
Comment (Koenkk) [inline src/devices/tuya.ts:11613]: Please use the model number that is written on the device here.

Comment (motezuki) [reply src/devices/tuya.ts]: Yes it is, I will modify the model accordingly

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `ZY_HPS01.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by motezuki** [comment]
Sure!
Sending the pull request


Em dom., 15 de set. de 2024 às 15:58, Koen Kanters ***@***.***>
escreveu:

**Pull request merged**